planning/planning_validator/include/planning_validator/planning_validator.hpp

```diff
@@ -91,7 +91,11 @@ class PlanningValidator : public rclcpp::Node
   rclcpp::Publisher<visualization_msgs::msg::MarkerArray>::SharedPtr pub_markers_;
 
   // system parameters
-  bool use_previous_trajectory_on_invalid_ = true;
+  enum class InvalidTrajectoryHandlingType {
+    PUBLISH_AS_IT_IT,
```

Comment:
typo: PUBLISH_AS_IT_**IS**?

Reply:
it was typo. THanks. Now fixed to as_it_is